README.md

```diff
@@ -175,6 +172,46 @@ Sets the volume mute state. Returns a promise.
 await lgtv.setVolumeMute(false);
 ```
 
+### `.getMacAddress(type: 'wired' | 'wifi')`
```

Comment:
Can you move those functions in the documentation so they are in alphabetical order?